contributing_to_docs/term_glossary.adoc

```diff
@@ -423,16 +423,13 @@ When referencing as a prerequisite for a procedure module, use the following
 construction: Install the OpenShift CLI (`oc`).
 
 ''''
-=== OpenShift control plane (also known as master)
+=== OpenShift control plane
 
 Usage: OpenShift control plane
 
-Provides a REST endpoint for interacting with the system and manages the state
-of the system, ensuring that all containers expected to be running are actually
-running and that other requests such as builds and deployments are serviced.
-New deployments and configurations are created with the REST API, and the state
-of the system can be interrogated through this endpoint as well.  An OpenShift
-control plane comprises the API server, scheduler, and SkyDNS.
+The control plane, which is composed of control plane machines, manages the {product-title} cluster. The control plane machines manage workloads on the compute machines, which are also known as worker machines.
```

Comment:
You could be more explicit about the differentiation between control plane and compute machines. The control plane manages the core cluster components. Applications are deployed on compute nodes. There are components on both sets of machines that enable this.